### PR TITLE
first api not worked fixed

### PR DESCRIPTION
**Pull request by jaimeen-s-simformsolutions**

### CODE REVIEW

File: src/components/login/Login.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import "../../App.css";
 import TextField from "@material-ui/core/TextField";
 // import { useStyles } from "./styles";
 import Box from "@material-ui/core/Box";
 import Snackbar from "@material-ui/core/Snackbar";
-import { privateLogin } from "../../utils/Index";
+import { privateLogin , privateLogout } from "../../utils/Index";
 
 function Login() {
   // const classes = useStyles();
```
Comment (Pratyush-Simform) [inline src/components/login/Login.js:13]: remove unnecessary comments

File: src/components/login/Login.js [hunk 2]
```diff
@@ -86,7 +86,10 @@ function Login() {
 
   if(localStorage.getItem('subDomain') === "Deny"){
     alert('You have Not Access for PatronPay\nPlease contact PatronPay Organization for the Access');
```
Comment (Pratyush-Simform) [inline src/components/login/Login.js:88]: Try to show pop up in place of alerts

File: src/components/login/Login.js [hunk 3]
```diff
@@ -86,7 +86,10 @@ function Login() {
...
-    localStorage.removeItem('subDomain');
+    privateLogout();
+    localStorage.removeItem("subDomain");
+    history.push("/");
+    // localStorage.removeItem('subDomain');
```
Comment (Pratyush-Simform) [inline src/components/login/Login.js:92]: Remove unnecessary comments

File: src/services/authenticationApi.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+export const passwordReset = async (email) => {
+  let subDom = localStorage.getItem("subDomain");
+  const api = `https://${subDom}/api/forgot_password/`;
+  const response = await Axios.post(api, {
+    email: email,
+  });
+  return response;
+};
+
+// export const getConfigApi = async () => {
```
Comment (Pratyush-Simform) [inline src/services/authenticationApi.js:37]: If this code is not useful remove the commented codes.

**Review by Pratyush-Simform — CHANGES_REQUESTED** [review]

File: src/components/login/Login.js (hunk 1 above)
Comment (jaimeen-s-simformsolutions) [reply src/components/login/Login.js]: solved

File: src/services/authenticationApi.js (hunk 4 above)
Comment (jaimeen-s-simformsolutions) [reply src/services/authenticationApi.js]: solved

**Review by Pratyush-Simform — APPROVED** [review]

**Pull request merged**